File: p2-python-hola-mundo/bicicleta.py
class Bicicleta:
    def __init__(self, marca, modelo, categoria, color):
        self.marca = marca
        self.modelo = modelo
        self.categoria = categoria
        self.color = color
    
    def mostrar(self):
        print(f"+++++++++++++++++++++++")
        print(f"Marca Bicicleta: {self.marca}")
        print(f"Modelo Bicicleta: {self.modelo}")
        print(f"Tamaño Bicicleta: {self.categoria}")
        print(f"Color Bicicleta: {self.color}")

File: p3-java/practicaRaulP3/Bicicleta.java
package practicaRaulP3;

public class Bicicleta {
    public String marca;
    public String modelo;

    public Bicicleta(String marca, String modelo) {
        this.marca = marca;
        this.modelo = modelo;
    }

    public void mostrar() {
        System.out.println("=================================");
        System.out.println("Marca: " + this.marca);
        System.out.println("Modelo: " + this.modelo);
        System.out.println("=================================");
    }
}